cijoe/scripts/fio_latency.py

```diff
@@ -336,50 +329,58 @@ def main(args, cijoe: Cijoe, step: Dict[str, Any]):
 
     err = 0
     try:
-        for (
-            (eng, setup),
-            bs,
-            io_depth,
-        ) in product(io_engines.items(), io_sizes, io_depths):
-            engine = determine_engine(eng, setup, devices, cijoe)
-            engine.prepare()
-
-            output_path = (
-                artifacts / f"fio-output_BS={bs}_IODEPTH={io_depth}_"
-                f"LABEL={engine.name_id}_{repetitions}_"
-                f"GROUP={engine.group}.txt"
-            )
-
-            # Run fio
-            fio = FIO(
-                bin=fio_bin,
-                engine=engine,
-                devices=devices,
-                name=f"{engine.name_id}",
-                rw="randread",
-                size=LATENCY_TEST_SIZE,
-                bs=bs,
-                repetitions=repetitions,
-                io_depth=io_depth,
-                numjobs=repetitions,
-                output_format="json",
-                time_based="1",
-                runtime="10",
-                eta_newline="1",
-                ramp_time="5",
-                norandommap="1",
-                thread="1",
-            )
-            cmd = " ".join(fio.cmd())
-
-            log.info(f"Executing the command: {cmd}")
-
-            err, state = cijoe.run(cmd, env=fio.env())
-            if err:
-                log.error(f"failed: {state}")
-
-            # Save the output to a file in artifacts directory
-            shutil.copyfile(state.output_fpath, output_path)
+        for run in runs:
+            print("running with:", run)
+
+            iosizes = run["iosizes"]
+            if isinstance(iosizes, str):
+                iosizes = ast.literal_eval(iosizes)
+            iodepths = run["iodepths"]
+            if isinstance(iodepths, str):
+                iosizes = ast.literal_eval(iodepths)
```

Comment:
This seems like it should be moved to cijoe.
In my opinion cijoe should support lists in the config out-of-the-box